alignak/objects/contact.py

```diff
@@ -202,11 +238,13 @@ def want_service_notification(self, notifways, timeperiods, downtimes,
         if not self.service_notifications_enabled:
             return False
 
-        # If we are in downtime, we do nto want notification
+        # If we are in downtime, we do not want notification
         for downtime_id in self.downtimes:
             downtime = downtimes[downtime_id]
             if downtime.is_in_effect:
+                self.in_scheduled_downtime = True
```

Comment:
Hmmm ... why not, but this needs some more tests that cannot be made with unit tests :/P Let's leave this as is ... and we will see.